django/db/backends/mysql/features.py

```diff
@@ -118,3 +118,9 @@ def ignores_table_name_case(self):
     def supports_default_in_lead_lag(self):
         # To be added in https://jira.mariadb.org/browse/MDEV-12981.
         return not self.connection.mysql_is_mariadb
+
+    @cached_property
+    def supports_json(self):
```

Comment:
Please also add this in `django/db/backends/base/features.py`. Perhaps the default value should be `False`? In `3.0.txt` this flag should also be documented.

Reply:
I've added it to `base`. I'm not sure how to document the flag in `3.0.txt`, though. I don't know how I should start documenting this whole thing, either. I haven't seen a note about a new database feature flag on the existing release notes, only about deprecated features. Perhaps I should start by describing `JSONField` under `What's new in Django 3.0`, below `ASGI support`?

Reply:
Yes, put it in the top section, as it's a major new feature.